content/develop/interact/search-and-query/advanced-concepts/aggregation-syntax.md

```diff
@@ -0,0 +1,193 @@
+
+`GROUPBY` ... `REDUCE` ... `APPLY` ... `GROUPBY` ... `REDUCE`
+
+{{< note >}}
+The examples on this page are based on a hypothetical "products" data set, which you can [download here](./data/products.txt).
+{{< /note >}}
+
+## Syntax and expression ordering
+
+The `FT.AGGREGATE` command processes multiple expressions in a pipeline. Below is the recommended order:
+
+1. `index` – the name of your index, which must be the first argument.
+1. `query` – your query, which must be the second argument.
+1. `FILTER` – filters raw documents before transformations or aggregation.
+1. `LOAD` – loads additional document attributes.
+1. `APPLY` – applies transformations on fields.
+1. `GROUPBY` – groups results by specific fields.
+1. `REDUCE` – performs aggregations. For example, `SUM`, `COUNT`, and `AVG`.
+1. `SORTBY` – orders the results based on specified fields.
+1. `LIMIT` – restricts the number of results returned.
+1. `DIALECT 2` - provides for more comprehensive syntax, for example using parameters in `FILTER` expressions.
+
+Other keywords will be discussed toward the end of this page.
+
+## When to use `@`
+
+You must add `@` at the start of a field name in the following circumstances:
+
+- When referencing fields loaded from documents. In the following example, `price` is a document field and must be prefixed with `@`.
+
+```sh
+FT.AGGREGATE products "*"
+  LOAD 1 @price
+  APPLY "@price * 1.1" AS adjusted_price
+  SORTBY 2 @adjusted_price DESC
+  LIMIT 0 10
+```
+
+- When referencing fields inside a `FILTER` clause that were loaded from documents.
+
+```sh
+FT.AGGREGATE products "*"
+  LOAD 1 @rating
+  FILTER "@rating >= 4.5"
+  LIMIT 0 10
+```
+
+- When referencing fields inside `GROUPBY` or `REDUCE` clauses.
+
+```sh
+FT.AGGREGATE products "*"
+  GROUPBY 1 @category
+  REDUCE SUM 1 @price AS total_price
+  LIMIT 0 10
+```
+
+- When referencing fields created by `REDUCE` in an `APPLY` or `FILTER` clauses.
+
+```sh
+FT.AGGREGATE products "*"
+  GROUPBY 1 @category
+  REDUCE SUM 1 @price AS total_price
+  APPLY "@total_price * 1.2" AS boosted_price
+  FILTER "@total_price > 1000"
+  LIMIT 0 10
+```
+
+- When referencing fields created by `APPLY` in another `APPLY` or `FILTER` clause.
+
+```sh
+FT.AGGREGATE products "*"
+  LOAD 2 @price @discount
+  APPLY "@price - @discount" AS net_price
+  APPLY "@net_price * 1.1" AS marked_up
+  FILTER "@net_price > 200"
+  LIMIT 0 10
+```
+
+- When referencing fields created by `APPLY` in a `SORTBY` clause.
+
+```sh
+FT.AGGREGATE products "*"
+  LOAD 2 @price @discount
+  APPLY "@price - @discount" AS net_price
+  SORTBY 2 @net_price DESC
+  LIMIT 0 10
+```
+
+## GROUPBY with multiple REDUCE operations
+
+`GROUPBY` can be followed by multiple `REDUCE` operations for different aggregations.
+
+```sh
+FT.AGGREGATE products "*"
```

Comment:
I think it would be nice to show the results of the queries you mention here, so that it is clearer what each operation does, and how the result looks like when applying it.